### PR TITLE
Add notes we took for the meeting today

### PR DESCRIPTION
**Pull request by saulshanabrook**
Copied notes from \[our hackmd\](https://hackmd.io/YaL0cHQRSNmEziJvx0x9Uw?edit) to the markdown file so they will be on the website!

Thanks everyone for coming, I had a blast :D

### CODE REVIEW

File: meetings/3_02_2021.md [hunk 1]
```diff
@@ -1,11 +1,57 @@
...
+    * Charlotte: How do we add ourselves to the github?
+* Quinten demo crazy file!
+    * Pull requests: We are gonna have pulls require one review on main page (and other pages?)
+    * Nina: How do we approve them?
+      * We now have enabled pull request reviews so every one needs to be reviewed by someone elseWe all went through and made or accepted pull requests
+* Saul Demo JupyterBook website
+* Breakout rooms where we try to edit things
+* For next time
+* naming session for our "github organization" and site?
+* Meeting time?
```
Comment (adelaide101) [inline meetings/3_02_2021.md:45]: I'm busy 9-5pm Monday-Fri & 10am-2pm Sat. so preferably not then!

File: meetings/3_02_2021.md [hunk 2]
```diff
@@ -1,11 +1,57 @@
...
+  * Nina will when2meet
+      * Quinten: like the meetings before the wednesday meeting
+      * monday tuesday!
+* Kali send zoom link!
+    * https://foodcorps.zoom.us/j/82728955419
+* Kali: An hour or hour and a half?
+  * Q: I like the flexability. 45 minutes, up to an hour and a half?
+    * I could see how small breakout could be like study group
+  * Q: bell ringers, breakout rooms in the begining!  
+* See you all tomorrow!
```
Comment (adelaide101) [inline meetings/3_02_2021.md:57]: sweet meeting, thank you :3

File: meetings/3_02_2021.md (hunk 1 above)
Comment (saulshanabrook) [reply meetings/3_02_2021.md:45]: Is that EST?

Comment (adelaide101) [reply meetings/3_02_2021.md:45]: yep, EST :)

**Comment by saulshanabrook** [comment]
I only remembered to take a screenshot after @adelaide101 left!



Does anyone else have any?

**Review by saracoderbrown — APPROVED** [review]

**Pull request merged**